lib/try_hook.rb

```diff
@@ -29,20 +29,17 @@ def goal_separator
   end
 
   def to_structured_results(_file, result, status)
-    /#{query_separator}
-?(.*)
-#{goal_separator}
-?(.*)
-/m =~ result
+    query_result = result[/#{query_separator}\n\K.*?(?=\n#{goal_separator}|\z)/m]
```

Comment:
Oops! So that basically means that we were not supporting results where there was no goal output - because the query failed -, right?

I fear I might have done the same mistake in the other interactive hooks :sweat_smile:

Reply:
Most likely, yes!

Technically it depends on how the worker behaves when it runs into an error, but usually you'd expect it to stop evaluating, right?